src/System.Drawing.Design.Primitives/src/SR.cs

```diff
@@ -0,0 +1,25 @@
+using System.Resources;
```

Comment:
This file needs to be deleted.

Reply:
Right. I remember doing it. will make sure that happen with next commit.